debug/kubeconfig

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUM1ekNDQWMrZ0F3SUJBZ0lCQURBTkJna3Foa2lHOXcwQkFRc0ZBREFWTVJNd0VRWURWUVFERXdwcmRXSmwKY201bGRHVnpNQjRYRFRJeU1URXpNREV6TVRnME4xb1hEVE15TVRFeU56RXpNVGcwTjFvd0ZURVRNQkVHQTFVRQpBeE1LYTNWaVpYSnVaWFJsY3pDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBT0E1ClRjNVRpeDZnZVU0RDVjRU4weXlRTHBpUXF3cDI4bkQ0NmZRczNNTG5xakdKWEk3TGk0Nmw0YmNhTlBSUTA2ZjgKdkNwNkc4eE9PRDNqOGkzUm9xbVdza1pKMzRVdnEybml6RHhRcW5IcnFGZzFJenJUZVRYNGVudjcwRnRtV0V0NQpUR2Q4U1dwRHlTVndxaWl5cWN5SDhFdUE4QWgrWGRRWEd4ajBzSEVkSTdsK1BxbndrczBSY3FZdnkxdmFuYzQwCitDaVhnUzV2b2dKblFVcFQwZjdBdFhrb1RUTjNJNmJnRkE2Q2Z2N3dST0JQSWtWdjVycFZMWHBOWnp4NE9SZzIKK0hXajhrSnVua0NuMFNZTUlnbUpONktzNVd1M2NsQjQvRXVQUUFOamNNUFR0djFXVm40YkFZR1RUdU5wZjlJMgpCaFRvZ3lRS3laYThvYW9lRFA4Q0F3RUFBYU5DTUVBd0RnWURWUjBQQVFIL0JBUURBZ0trTUE4R0ExVWRFd0VCCi93UUZNQU1CQWY4d0hRWURWUjBPQkJZRUZFK3Y3N0xpOVRDTXJvYlkvalVNcHh5aGdueklNQTBHQ1NxR1NJYjMKRFFFQkN3VUFBNElCQVFCZ09JZzFkeEt3OTg1aFVNNGtBRUVHZXdwc0c4S0k2KzBwTW5acjJEeDN0RGlhYjA5KwpCeXNyVXp2QXlFd1pDeTdLRVpmdTU5N0gzV0lkdDdOaTU1QkEvbEJCOVN5L0I0NG5KeVVvcEpjbHgycWNJbmJjCmtTWE9tSm80WHZ6c1VENDJPZHZwR21KUzdnOVF5NW5BTnFVK1dhRzZ3aytySURObWd2SDJzbzFSdEZtRzFQQmEKTHhzSTRhTHdEMCttcVozLytUT2JMTDdSUzFVZHJJaXZkKzh1NjNxTGlSOVV3blFoSFh6Q21zOStvS0k2dVZORwovZGJEak9vU3lTMjZDM0lRU3ZhTXlYUm9vTkRJb3VkWlVpM3E4d2N3QnIrbDZaRVBOa1M5MlJVeDJ1d28zZm1IClhzbkFvajU1Ulg2L2JkQVhoYVRWcllDRGlVbTZLODVpV1llVwotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
+    server: https://012B84B4168EAFFF3D20278FF4DEA47B.gr7.eu-west-1.eks.amazonaws.com
+  name: cluster-uxpeks
+contexts:
+- context:
+    cluster: cluster-uxpeks
+    user: cluster-uxpeks
+  name: cluster-uxpeks
+current-context: cluster-uxpeks
+kind: Config
+preferences: {}
+users:
+- name: cluster-uxpeks
+  user:
+    token: k8s-aws-v1.aHR0cHM6Ly9zdHMuZXUtd2VzdC0xLmFtYXpvbmF3cy5jb20vP0FjdGlvbj1HZXRDYWxsZXJJZGVudGl0eSZWZXJzaW9uPTIwMTEtMDYtMTUmWC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBUlU1SkZMVTZGTEI1Q0U1NiUyRjIwMjIxMTMwJTJGZXUtd2VzdC0xJTJGc3RzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMjExMzBUMTUwMzIwWiZYLUFtei1FeHBpcmVzPTkwMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QlM0J4LWs4cy1hd3MtaWQmWC1BbXotU2lnbmF0dXJlPWJlMDc2NjgxMjhjMTExMjJlNTljYjI0NWVlM2YyNDQ3NzIyMTA5NzA2N2E4ZDU1NjQ2OTgwNjE2N2U0ZmE5MTM
```

Comment:
Umm I think that's a secret inside Git. I know it's probably just some temporary cluster but still that should be inside some .gitignore

Reply:
good point. Cluster was deleted but definitely it shuld not be there.